src/static/css/styles.css

```diff
@@ -39,17 +39,35 @@ a {
     text-decoration: none;
 }
 
-a:hover {
+section a:hover {
     background-color: #fff;
     border-color: #667a7d;
     color: #667a7d;
 }
 
-@media (min-width: 800px) {
-    img {
-        margin-left: -20px;
-    }
+.svg {
+    display: block;
+    box-shadow: 1px 1px 14px rgba(0,0,0,0.2);
+}
+
+.language-switcher {
+    width: 60px;
+    top: 20px;
+    right: 40px;
+    position: absolute;
+    cursor: pointer;
+}
 
+.language-switcher__dropdown {
+    display: none;
+    margin-top: 10px;
+}
+
+.language-switcher:hover .language-switcher__dropdown {
```

Comment:
I have two concerns with displaying the dropdown on hover: there is no hover on mobile, and it's not discoverable to screen readers. I tried to switch languages using only my keyboard and it wasn't possible.

Could you try implementing this using native UI elements like `<select>` and `<option>`? This should address both of the usability concerns above.

Reply:
It would be difficult to include images inside native elements like `select` and `option` for the dropdown menu. Some references:
1. [putting-images-with-options-in-a-dropdown-list](https://stackoverflow.com/questions/4941004/putting-images-with-options-in-a-dropdown-list)
2. [how-to-add-small-thumbnails-to-option-elements-in-a-select-list/31697569](https://stackoverflow.com/questions/31697386/how-to-add-small-thumbnails-to-option-elements-in-a-select-list/31697569).

Emoji flags would be used in place of images.